testmore/newFuncs/script.R

```diff
@@ -0,0 +1,140 @@
+## Testing some new functions and modifications
```

Comment:
This is a great test file, well structured and thorough - exactly as tests should be
It does not work for me though, it breaks on errors and does not produce anything useful.

I am wondering if you have some other options than me in your R profile that transforms errors to printed messages.

Reply:
I run the tests with `R CMD BATCH --slave --no-timing script.R res.EXP`, no extra settings in my profile or environment

Reply:
the newFuncs tests did not sink results into `res.out` file.

Reply:
bug fix of `robust` and `NA_SMA` testmore scripts in commit 3068325. `newFuncs` tests untouched.